tests/integration/workspace_access/test_clusters.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from datetime import timedelta
+
+from databricks.labs.blueprint.tui import MockPrompts
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+
+from databricks.labs.ucx.workspace_access.clusters import ClusterAccess
+
+logger = logging.getLogger(__name__)
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_prepare_environment(ws, make_cluster):
```

Comment:
integration test is probably not required for this :)

Reply:
Removed Integration test